docs/dev/tools/cli.md

```diff
@@ -716,8 +716,11 @@ viam module upload --version=1.0.0 --platform=darwin/arm64 packaged-module.tar.g
 | `--binary` | The binary for the module to run. The binary has to work on the OS or processor of the device. | `update-models` | **Required** |
 | `--count` | Number of cloud builds to list, defaults to displaying all builds | `build list` | Optional |
 | `--force` | Skip local validation of the packaged module, which may result in an unusable module if the contents of the packaged module are not correct | `upload` | Optional |
-| `--id` | The build ID to list or show logs for, as returned from `build start` | `build list`, `build logs` | Optional |
+| `--home` | Specify home directory for a remote machine where `$HOME` is not the default `/root` | `reload` | Optional |
+| `--id` | The build ID to list or show logs for, as returned from `build start` | `build list`, `build logs`, `reload` | Optional |
```

Comment:
We're not 100% consistent in adding the period at the end across this long file, but let's be consistent with the content we're adding:
```suggestion
| `--home` | Specify home directory for a remote machine where `$HOME` is not the default `/root`. | `reload` | Optional |
| `--id` | The build ID to list or show logs for, as returned from `build start`. | `build list`, `build logs`, `reload` | Optional |
```